lib/features/email/presentation/widgets/attachment_item_widget.dart

```diff
@@ -25,14 +27,15 @@ class AttachmentItemWidget extends StatelessWidget {
     Key? key,
     required this.attachment,
     this.downloadAttachmentAction,
+    this.viewAttachmentAction,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return Material(
       color: Colors.transparent,
       child: InkWell(
-        onTap: () => downloadAttachmentAction?.call(attachment),
+        onTap: () => viewAttachmentAction?.call(attachment),
```

Comment:
So, where is `downloadAttachmentAction` used?

Reply:
Line 90 same file